Components/product-listing.css

```diff
@@ -0,0 +1,60 @@
+.main-content{
+    width: 82%;
+}
+.product-container{
+    height: 25rem;
+    width: 69rem;
+    border: 1px solid var(--color-gray-200);
+    margin-top:5px ;
```

Comment:
suggestions - try
margin : 5px auto;
for product-container to horizontally center it in main-content